src/gui/SettingsWidgetGeneral.ui

```diff
@@ -351,6 +351,58 @@
         </layout>
        </item>
        <item>
+        <layout class="QHBoxLayout" name="autoTypeDelayHorizontalLayout">
+         <property name="spacing">
+          <number>15</number>
+         </property>
+         <property name="sizeConstraint">
+          <enum>QLayout::SetDefaultConstraint</enum>
+         </property>
+         <item alignment="Qt::AlignLeft">
+          <widget class="QLabel" name="autoTypeDelayLabel">
+           <property name="text">
+            <string>Auto-Type delay</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QSpinBox" name="autoTypeDelaySpinBox">
```

Comment:
It doesn't have one because it's the only option like that on this tab. But as soon as you have two or more, you need a grid t align them. So yes, please wrap those two options in a grid.